qutip/solve/_stochastic.pyx

```diff
@@ -1916,35 +1916,36 @@ cdef class SMESolver(StochasticSolver):
                                         np.ndarray[complex, ndim=1] guess) except *:
         # np.ndarray to memoryview is OK but not the reverse
         # scipy function only take np array, not memoryview
-        spout, check = sp.linalg.bicgstab(self.imp(t, data=1).as_scipy(),
-                                          dvec, x0=guess, tol=self.tol, atol=1e-12)
+        spout, check = sp.linalg.bicgstab(self.imp(t).data.as_scipy(),
```

Comment:
This still has the unsafe assumption that `data` will always be `CSR` I think?

Reply:
I made the conversion when creating `self.imp` in `stochastic.py`. So unless someone try to use this without using the python interface it should be safe.